falco/values.yaml

```diff
@@ -348,6 +399,8 @@ falco:
     - /etc/falco/falco_rules.yaml
     - /etc/falco/falco_rules.local.yaml
     - /etc/falco/rules.d
+    - /etc/falco/falcoctl-rules
+
```

Comment:
This should not work as expected because the `falco_rules.yaml` will be downloaded in another folder, and as result, Falco will crash (rules will be loaded twice).

Moreover, this approach is fragile (see one of the comments above, where I wonder how we can enforce `/etc/falco/falcoctl-rules`).

That being said, I don't have a solution yet :sweat_smile:

Reply:
Side note: order matters here. To improve the UX, we should also pay attention to loading custom user-provided rules, if any, always _after_ those already installed.